app/src/main/java/com/cornellappdev/uplift/ui/screens/MainReminderScreen.kt

```diff
@@ -0,0 +1,113 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Icon
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarColors
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.uplift.R
+import com.cornellappdev.uplift.util.GRAY00
+import com.cornellappdev.uplift.util.GRAY01
+import com.cornellappdev.uplift.util.LIGHT_GRAY
+import com.cornellappdev.uplift.util.montserratFamily
+import com.google.accompanist.systemuicontroller.SystemUiController
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+
+/**
+ * The home page of Uplift.
+ */
+@OptIn(ExperimentalPermissionsApi::class, ExperimentalMaterial3Api::class)
+@Composable
+fun MainReminderScreen() {
+    val systemUiController: SystemUiController = rememberSystemUiController()
+
+    systemUiController.isStatusBarVisible = false
+    systemUiController.isNavigationBarVisible = false // Navigation bar
+    systemUiController.isSystemBarsVisible = false
+
+    Scaffold(topBar = {
+        TopAppBar(
+            title =  {Text(
+                text = "Reminders",
+                fontSize = 24.sp,
+                fontFamily = montserratFamily,
+                fontWeight = FontWeight.Bold) },
+            colors = TopAppBarDefaults.topAppBarColors(containerColor = LIGHT_GRAY),
+            navigationIcon = {
+                IconButton(onClick = {}) {
+                    Icon(painter = painterResource(R.drawable.backarrow), contentDescription = "")
+                }
+            }
+
+        ) }) {
+        innerPadding ->
+        Column(modifier = Modifier.padding(innerPadding)) {
+            Column(modifier = Modifier.padding(horizontal = 24.dp)) {
+                ReminderListItem("Class Reminders", R.drawable.classes, {})
+                HorizontalDivider(color = GRAY01)
+                ReminderListItem("Workout Reminders", R.drawable.gym_simple, {})
+                HorizontalDivider(color = GRAY01)
+                ReminderListItem("Capacity Reminders", R.drawable.capacities, {})
+            }
+        }
+    }
+}
+
+@Composable
+private fun ReminderListItem(txt: String, resourceId: Int, onClick: () -> Unit) {
+    Row(verticalAlignment = Alignment.CenterVertically, horizontalArrangement = Arrangement.SpaceBetween, modifier = Modifier.height(72.dp).fillMaxWidth()) {
+        Row(horizontalArrangement = Arrangement.spacedBy(8.dp)) {
```

Comment:
This might be a bit nitpicky, but I think the icon image and the text are not centered

Reply:
Good catch! Definitely not centered right now. Make sure both rows align center vertically.